index.bs

```diff
@@ -753,6 +753,18 @@ To <dfn>find a matching trigger spec</dfn> given an [=attribution source=]
 
     </dl>
 
+<h3 id="destination-limit-algorithm-header">Destination limit algorithm</h3>
+
+A <dfn>destination limit algorithm</dfn> is one of the following:
+
+<dl dfn-for="destination limit algorithm">
+: "<dfn><code>fifo</code></dfn>"
+:: Counts the highest prioritized and most recent [=attribution source/attribution destinations=].
+: "<dfn><code>lifo</code></dfn>"
+:: Counts the earliest [=attribution source/attribution destinations=].
```

Comment:
Are `fifo` and `lifo` opposites of each other in this scheme? If not, the naming is a bit misleading.

Reply:
It's not strictly opposite, fifo uses priority but lifo is not.

Reply:
I wonder if we should have some more precise names, then. IIUC, in `fifo`, sources are sorted by priority and *then* by recency, so maybe it should be `priority-fifo`?

And should it be an error to specify the `priority` field if the algorithm is `lifo`? At the very least we should probably warn about this in the header validator.

Reply:
Yeah renaming to priority-fifo SGTM. The priority can still be set if the algorithm is lifo, and will be stored as is for future fifo use.

Reply:
Not needed anymore.